e2e/scenarios/open-allocations.test.ts

```diff
@@ -0,0 +1,52 @@
+import { expect } from 'chai'
+import hre from 'hardhat'
+import { getIndexerFixtures, IndexerFixture } from './fixtures/indexers'
+
+enum AllocationState {
+  Null,
+  Active,
+  Closed,
+  Finalized,
+  Claimed,
+}
+
+let indexerFixtures: IndexerFixture[]
+
+describe('Open allocations', () => {
+  const { contracts, getTestAccounts } = hre.graph()
+  const { GraphToken, Staking } = contracts
+
+  before(async () => {
+    indexerFixtures = getIndexerFixtures(await getTestAccounts())
+  })
+
+  describe('GRT balances', () => {
+    it(`indexer balances should match airdropped amount minus staked`, async function () {
+      for (const indexer of indexerFixtures) {
+        const address = indexer.signer.address
+        const balance = await GraphToken.balanceOf(address)
+        expect(balance).gte(indexer.grtBalance.sub(indexer.stake))
```

Comment:
Should this be `.eq` or is this assuming some indexers might start with nonzero balance? (In which case the description of the test seems off?)

Reply:
Yes good catch. I think I changed this to run the tests against mainnet and forgot to change back. 
Eventually it would be nice to have this scenarios so we could run them against any network, but there is the previous state thing which we would need to account for, which is something for another day.